kickstarts/partials/main/disk_layout.ks.erb

```diff
@@ -8,5 +8,10 @@ part /var/www/miq_tmp --ondrive=vda                     --size=10240 --fstype=xf
 part pv.2             --ondrive=vda                     --size=10240 --grow
 
 volgroup vg_system    --pesize=4096 pv.1
-logvol /              --name=lv_os   --vgname=vg_system --size=10240 --fstype=xfs
-logvol swap           --name=lv_swap --vgname=vg_system --size=4096  --grow
+logvol /              --name=lv_os             --vgname=vg_system --size=4680 --fstype=xfs
```

Comment:
Is this too small to contain the database?  I think we ship with a 40GB disk, right?